File: 217. Contains Duplicate/217. Contains Duplicate.php
class Solution {

    /**
     * @param Integer[] $nums
     * @return Boolean
     */
    function containsDuplicate($nums) {
        $numsMap = [];
        foreach($nums as $num){
            if(isset($numsMap[$num]))
                return true;
            $numsMap[$num] = 1;
        }
        return false;
    }
}

File: 217. Contains Duplicate/217. Contains Duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        HashSet<Integer> numMap = new HashSet<Integer>();

        for(int i = 0; i < nums.length; i++) {
            if (numMap.contains(nums[i])) return true;
            numMap.add(nums[i]);
        }

        return false;
    }
}